src/wp-includes/functions.php

```diff
@@ -9090,3 +9092,151 @@ function wp_is_heic_image_mime_type( $mime_type ) {
+
+	/**
+	 * Filters $icon_url before the tooltip is rendered in the HTML.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @param string $icon_url The URL to the icon to be used for this tooltip.
+	 *                         * Pass a base64-encoded SVG using a data URI, which will be colored to match
+	 *                         the color scheme. This should begin with 'data:image/svg+xml;base64,'.
+	 *                         * Pass the name of a Dashicons helper class to use a font icon,
+	 *                         e.g. 'dashicons-editor-help'.
+	 *                         * Pass 'none' to leave span.wp-tooltip-button-span empty so an icon can be added via CSS.
+	 * @param string $field_id The unique identifier for the tooltip container.
+	 */
+	$icon_url = apply_filters( 'wp_tooltip_icon', $icon_url, $field_id );
+	$icon_url = set_url_scheme( $icon_url );
+
+	if ( str_starts_with( $icon_url, 'dashicons-' ) ) {
+		$icon      = 'class="dashicons ' . sanitize_html_class( $icon_url ) . '"';
+		$icon_html = '<span ' . $icon . ' aria-hidden="true"></span>';
+	} elseif ( str_starts_with( $icon_url, 'data:image/svg+xml;base64,' ) ) {
+		$icon      = ' style="background-image:url(\'' . esc_attr( $icon_url ) . '\')"';
+		$icon_html = '<svg height="20" width="20" aria-hidden="true" focusable="false"' . $icon . '></svg>';
+	} elseif ( str_starts_with( $icon_url, 'http' ) ) {
+		$icon_html = '<img height="20" width="20" src="' . esc_url( $icon_url ) . '" alt="" />';
+	} elseif ( 'none' === $icon_url ) {
+		$icon_html = '';
+	} else {
+		_doing_it_wrong(
+			__FUNCTION__,
+			sprintf(
+				/* translators: %s: Invalid $icon_html passed in 'pre_wp_tooltip_rendered' hook. */
+				__( 'The passed "%1$s" argument in the "%2$s" filter is invalid.' ),
+				'$icon_html',
+				'pre_wp_tooltip_rendered'
+			),
+			'6.8.0'
+		);
+
+		$icon_html = '<span class="dashicons dashicons-editor-help" aria-hidden="true"></span>';
+	}
+
+	/**
+	 * Fires before the tooltip is rendered.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @param string $field_id     The unique identifier for the tooltip container.
+	 * @param string $tooltip_text The text content to be displayed in the tooltip.
+	 * @param string $button_label The label for the tooltip button.
+	 * @param string $position     The default position of the tooltip.
+	 * @param string $icon_url     The URL to the icon to be used for this tooltip.
+	 *                             Pass a base64-encoded SVG using a data URI, which will be colored to match
+	 *                             the color scheme. This should begin with 'data:image/svg+xml;base64,'.
+	 *                             Pass the name of a Dashicons helper class to use a font icon,
+	 *                             e.g. 'dashicons-editor-help'.
+	 *                             Pass 'none' to leave span.wp-tooltip-button-span empty so an icon can be added via CSS.
+	 */
+	add_action( 'pre_wp_tooltip_rendered', $field_id, $tooltip_text, $button_label, $position, $icon_url );
```

Comment:
What is this good for? `pre_` actions are not so common, and `pre_` filters would be used to short-circuit rendering.